firmware/powerboard2/code/app/LED_componentSpecific.c

```diff
@@ -22,13 +22,13 @@ const LED_config_S LED_config =
         {
             .pin = 7,
             .GPIOPort = GPIOB,
-            .initState = LED_STATE_ON,
+            .initState = LED_STATE_BLINK_NORMAL,
         },
         [LED_CHANNEL_GREEN] =
         {
             .pin = 0,
             .GPIOPort = GPIOB,
-            .initState = LED_STATE_BLINK_NORMAL,
+            .initState = LED_STATE_ON,

```

Comment:
Can you make the green LED blink instead? I changed this, sorry. Make the blue LED solid on.